flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/PendingCheckpoint.java

```diff
@@ -371,7 +371,7 @@ public TaskAcknowledgeResult acknowledgeTask(
 				acknowledgedTasks.add(executionAttemptId);
 			}
 
-			List<OperatorID> operatorIDs = vertex.getJobVertex().getOperatorIDs();
+			List<OperatorID> operatorIDs = vertex.getJobVertex().getOperatorIdPairList().getOperatorIds();
```

Comment:
`vertex.getOperatorIdPairList().getOperatorIds` sounds a bit repetitive to me.

How about `vertex.getOperatorIDs().getGeneratedIDs()`?

Reply:
done